src/modules/account/views/ViewAccount.vue

```diff
@@ -32,8 +30,7 @@ import AccountComponent from "@/modules/account/components/AccountComponent.vue"
 import AssetComponent from "@/modules/account/components/AssetComponent.vue";
 import NamespaceComponent from "@/modules/account/components/NamespaceComponent.vue";
 import MetadataComponent from "@/modules/account/components/MetadataComponent.vue";
-import MultisigComponent from "@/modules/account/components/MultisigComponent.vue";
-import SchemeComponent from "@/modules/account/components/SchemeComponent.vue";
+import MultisigComponent from "../components/MultisigComponent.vue";
```

Comment:
what's the difference between `@/modules/account/components/MultisigComponent.vue` and `../components/MultisigComponent.vue`

Reply:
It is basically the same but ../components/MultisigComponent.vue is short form of  @/modules/account/components/MultisigComponent.vue. I will change it.